docs/config.md

```diff
@@ -0,0 +1,38 @@
+# Adding an  extra category to the search bar
+
+# To start of here is an example config
+
+		"ImqsServerMirror": {
+			"Tables": {
+			"AssetComponent": {
+					"Categories": [
+						"Assets"
+					],
+					"FriendlyName": "Facility Name",
+					"Fields": [
+						{
+							"FriendlyName": "ComponentID",
+							"Field": "ComponentID",
+							"Order": "1"
+						},
+						{
+							"FriendlyName": "Facility Name",
+							"Field": "AssetFacilityName",
+							"Order": "2"
+						}
+					]
+				},
+
+1. In order to search on a new table it has to be entered into the config under the rightful DB it belongs to.
+# DBs contains tables:
```

Comment:
You should add another `#` here. It will make the title a bit smaller than then main title, as this is a sub section of the title.